docker-compose.travis.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.7"
+
+services:
+  backend:
+    image: humanconnection/nitro-web:builder
```

Comment:
Do you know the reason for having yet another `docker-compose.yml` just for travis? Because of shared folders. Travis struggles to delete them after running the tests, thus silently failing our deployments. The reason why Travis cannot delete the files is that the `docker` user on Travis creates the files, not the user who is later trying to delete them.